repo/packages/A/artifactory-primary/0/marathon.json.mustache

```diff
@@ -0,0 +1,90 @@
+  {{/artifactory-primary.high-availability.enabled}}
+   "container": {
+       "type": "DOCKER",
+       "docker": {
+           {{^artifactory-primary.high-availability.enabled}}
+           "image": "{{resource.assets.container.docker.e183c36bdb4b}}",
+           {{/artifactory-primary.high-availability.enabled}}
+           {{#artifactory-primary.high-availability.enabled}}
+           "image": "{{resource.assets.container.docker.71c42177cf14}}",
+           {{/artifactory-primary.high-availability.enabled}}
+           "network": "HOST",
+            "forcePullImage":true
+       },
+       "volumes": [
+          {{^artifactory-primary.high-availability.enabled}}
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/data",
+          "hostPath": "{{artifactory-primary.artifactory-home}}/{{artifactory-primary.framework-name}}/data",
+          "mode": "RW"
+          },
+          {
+             "containerPath": "/var/opt/jfrog/artifactory/logs",
+             "hostPath": "{{artifactory-primary.artifactory-home}}/{{artifactory-primary.framework-name}}/logs",
+             "mode": "RW"
+          },
+          {
+             "containerPath": "/var/opt/jfrog/artifactory/backup",
+             "hostPath": "{{artifactory-primary.artifactory-home}}/{{artifactory-primary.framework-name}}/backup",
+             "mode": "RW"
+          }
+          {{/artifactory-primary.high-availability.enabled}}
+          {{#artifactory-primary.high-availability.enabled}}
+          {
+             "containerPath": "/var/opt/jfrog/cluster",
+             "hostPath": "{{artifactory-primary.high-availability.cluster-home}}/cluster-home",
+             "mode": "RW"
+          },
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/data",
+          "hostPath": "{{artifactory-primary.high-availability.cluster-home}}/{{artifactory-primary.framework-name}}/data",
+          "mode": "RW"
+          },
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/logs",
+          "hostPath": "{{artifactory-primary.high-availability.cluster-home}}/{{artifactory-primary.framework-name}}/logs",
+          "mode": "RW"
+          },
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/backup",
+          "hostPath": "{{artifactory-primary.high-availability.cluster-home}}/{{artifactory-primary.framework-name}}/backup",
+          "mode": "RW"
+          }
+          {{/artifactory-primary.high-availability.enabled}}
+       ]
+   },
+  "labels": {
+      "HAPROXY_GROUP":"external"
+   },
+   "env": {
+    "DATABASE_CONNECTION_STRING": "{{artifactory-primary.high-availability.database.connection-string}}",
+    "DATABASE_USER":  "{{artifactory-primary.high-availability.database.user}}",
+    "DATABASE_PASSWORD":  "{{artifactory-primary.high-availability.database.password}}",
+    "ART_PRIMARY_LICENSE": "{{artifactory-primary.artifactory-primary-license}}",
```

Comment:
Just using `PRIMARY_LICENSE` instead of `ART_PRIMARY_LICENSE` would be cleaner here.